### PR TITLE
Swagger from file

### PR DESCRIPTION
**Pull request by ninit**
Hi,

This is a proposal to decode a special line in a docstring `swagger_from_file: path`.

I think it could be useful sometimes to take the yaml swagger spec out of the docstring:
- if you have other things to say in the docstring,
- if it's easier to edit yaml code from a .yml file.

In the pull request you'll find besides flask_swagger.py a modified example and README.md

Please criticize as needed.
Thanks.

### CODE REVIEW

File: flask_swagger.py [hunk 1]
```diff
@@ -88,7 +120,8 @@ def _extract_array_defs(source):
     return defs
 
 
-def swagger(app, prefix=None, process_doc=_sanitize, template=None):
+def swagger(app, prefix=None, process_doc=_sanitize,
+            from_file_keyword='swagger_from_file', template=None):
```
Comment (atlithorn) [inline flask_swagger.py]: I would prefer from_file_keyword to be default None in which case skip the find_from_file process

File: flask_swagger.py [hunk 2]
```diff
@@ -17,10 +17,42 @@ def _sanitize(comment):
...
+    with open(path) as f:
+        doc = f.read()
+    return doc
+
+
+def _parse_docstring(obj, process_doc, from_file_keyword):
     first_line, other_lines, swag = None, None, None
     full_doc = inspect.getdoc(obj)
     if full_doc:
+        from_file = _find_from_file(full_doc, from_file_keyword)
```
Comment (atlithorn) [inline flask_swagger.py]: Check for `from_file_keyword is None` in which case just skip the _find_from_file process

**Comment by atlithorn** [comment]
Otherwise looks good and thanks for contributing!

**Comment by ninit** [comment]
Thanks for your review.
You're right, it's better to leave this off by default.
I've just committed the change.

**Pull request merged**